SpatialGDK/Source/SpatialGDK/Public/Utils/SpatialLatencyTracer.h

```diff
@@ -96,6 +96,7 @@ class SPATIALGDK_API USpatialLatencyTracer : public UObject
 
 	bool IsValidKey(TraceKey Key);
 	TraceKey GetTraceKey(const UObject* Obj, const UFunction* Function);
+	TraceKey GetTraceKey(const UObject* Obj, const UProperty* Property);
```

Comment:
Nit: I would prefer a name that suggests that the function modifies the object. For example: GetAndRemoveTraceKey (not saying that's the one to use but...).